applications/asset_tracker/src/cloud_codec/cloud_codec.c

```diff
@@ -458,6 +458,32 @@ int cloud_encode_env_sensors_data(const env_sensor_data_t *sensor_data,
 	return cloud_encode_data(&cloud_sensor, output);
 }
 
+int cloud_encode_motion_data(const motion_data_t *motion_data,
+				  struct cloud_msg *output)
+{
+	__ASSERT_NO_MSG(motion_data != NULL);
+	__ASSERT_NO_MSG(output != NULL);
+
+	struct cloud_channel_data cloud_sensor;
```

Comment:
Does/could/should there exist an init function for cloud_channel_data ?

As this is written, if another field is added in the future, and this field is read by cloud_encode_data, then we might accidentally forget to initialize that field from this function and get undefined behaviour when cloud_encode_data reads it.

Reply:
I see your point. We will add this as a task to our backlog of improvements